quickstarts/multicodebase-hellos/README.md

```diff
@@ -0,0 +1,49 @@
+# Firebase SDK for Cloud Functions Quickstart - Multi-codebases
+
+This quickstart demonstrates how to setup a multi-codebase repository that allows you to setup multiple source directories to manage your function definitions. 
+
+
+## Setup
+
+This repository contains 2 different function sources:
```

Comment:
Will a codebase always correspond to a language, or is it more arbitrary?  

We may not need to talk about that here, but definitely would in the docs.

Reply:
It can be in the same language.

For example, say you have a function that requires a heavy initialization which affects the cold start time. We've previously recommended that users do lazy initialization (https://firebase.google.com/docs/functions/tips#do_lazy_initialization_of_global_variables) so the cost of initialization doesn't affect other functions in the file.

With codebases, you can separate out the function into different codebases so you can group functions that requires the heavy initialization (w/ corresponding impact on cold start) and ones that does not!